paper/literature.bib

```diff
@@ -385,103 +396,148 @@ @INPROCEEDINGS{1173142
     volume={},
     number={},
     pages={15-22},
-    doi={10.1109/INFVIS.2002.1173142}}
+    doi={10.1109/INFVIS.2002.1173142}
+}
 
-@misc{jsoneditoronlineJSONEditor,
-    author = {Jos de Jong},
-    title = {{J}{S}{O}{N} {E}ditor {O}nline: {J}{S}{O}{N} editor, {J}{S}{O}{N} formatter, query {J}{S}{O}{N} --- jsoneditoronline.org},
-    howpublished = {\url{https://jsoneditoronline.org}},
+@misc{mostpopularides,
+    author = {Pierre Carbonnelle},
+    title = {{T}{O}{P} {I}{D}{E} {T}op {I}ntegrated {D}evelopment {E}nvironment index --- pypl.github.io},
+    howpublished = {\url{https://pypl.github.io/IDE.html}},
     year = {},
-    note = {[Accessed 08-10-2023]},
+    note = {[Accessed 18-05-2023]},
 }
 
-@misc{xmlblueprintEditorXMLBlueprint,
-    author = {},
-    title = {{X}{M}{L} {E}ditor - {X}{M}{L}{B}lueprint --- xmlblueprint.com},
-    howpublished = {\url{https://www.xmlblueprint.com}},
-    year = {},
-    note = {[Accessed 08-10-2023]},
+@misc{githubBspEditor,
+    author = {Neubauer, F.},
+    title = {{G}it{H}ub - Logende/BossShopProEditor:  powerful and user-friendly web application, which makes setting up shops for the Bukkit plugin BossShopPro way easier. --- github.com},
+    howpublished = {\url{https://github.com/Logende/BossShopProEditor}},
+    year = {2023},
+    note = {[Accessed 11-10-2023]},
 }
-@misc{oxygenxmlCompleteSolution,
-    author = {},
-    title = {{T}he complete solution for {X}{M}{L} authoring, development and collaboration. --- oxygenxml.com},
-    howpublished = {\url{https://www.oxygenxml.com/xml_developer.html}},
-    year = {},
-    note = {[Accessed 08-10-2023]},
+
+@misc{ace-editor,
+    title = {Code Editor},
+    url = {https://ace.c9.io/},
+    author={Harutyun Amirjanyan, Fabian Jakobs},
+    year = {2010},
+    note={[Accessed 14-05-2023]}
 }
 
-@misc{altovaEditorXMLSpy,
-    author = {},
-    title = {{X}{M}{L} {E}ditor: {X}{M}{L}{S}py --- altova.com},
-    howpublished = {\url{https://www.altova.com/xmlspy-xml-editor}},
+@misc{ajv-validator,
+    title={A useful library for validating a json file based on the schema},
+    url={https://ajv.js.org/},
+    author={Evgeny Poberezkin etc.},
+    year={2015},
+    note={[Accessed 05-07-2023]}
+}
+
+@misc{Viotti_Lagoni_2023,
+    title={Sourcemeta/alterschema: Convert between JSON Schema Specification versions.},
+    url={https://github.com/sourcemeta/alterschema},
+    journal={GitHub},
+    author={Viotti, Juan Cruz and Lagoni, Jonas},
+    year={2023}
+}
+
+@misc{pyEnzyme,
+    title = {Lightweight syntax for rapid development of data management solution in enzymology and biocatalysis.},
+    url = {https://github.com/EnzymeML/PyEnzyme},
+    journal = {Github},
+    author = {Jan Range},
+    note = {[Accessed 15-10-2023]}
+}
+
+@misc{strenda,
+    title = {Schema provided by Jan Range},
```

Comment:
This citation is a bit weird @Logende , do we really need it?
If so, @beyond2void the Author must be formatted like this: "Range, Jan"

Reply:
I think it make sense to refer to the schema, but we should give the reference a more formal title